config/jail.conf

```diff
@@ -330,7 +347,7 @@ logpath = /opt/openhab/logs/request.log
 port    = http,https
 logpath = %(nginx_error_log)s
 
-# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module` 
+# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module`
```

Comment:
Thanks for this, I saw that before commit and thought "well, it'll probably happen in someones editor eventually, may as well trim them where they come up." But, I am happy to revert those changes.